platform/nativescript/runtime/directives/view.js

```diff
@@ -1,7 +1,8 @@
 export default {
-  inserted(el, { arg, modifiers }) {
+  inserted(el, binding, vnode, oldNode) {
     const parent = el.parentNode.nativeView
-
+    const arg = binding.rawName.split(":")[1];
+    const modifiers = binding.modifiers;
```

Comment:
@farfromrefug I'm assuming this is a related change to `platform/nativescript/compiler/modules/view.js`, can you elaborate why this is needed?

Reply:
it seems to have changed in Vue 2.6, your was not working anymore and i could not get the "arg" as before

Reply:
@farfromrefug I'm assuming it's this commit that changed directives: https://github.com/vuejs/vue/commit/dbc0582587f90e78867809bb6ae683301cd0626b

wonder if changing the order in `platform/nativescript/compiler/modules/view.js` for the arguments would make it work without this change.

Reply:
yes will try

Reply:
fix working as we talked about